packages/lib/Synchronizer.ts

```diff
@@ -524,6 +524,9 @@ export default class Synchronizer {
 					// await uploadSyncInfo(this.api(), remoteInfo);
 				}
 			} catch (error) {
+				if (error.message === 'Invalid authentication code') {
```

Comment:
Isn't Joplin Cloud setting an http status code, and can we use this instead? We shouldn't check error message as they can change in the future.